src/Components/header.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const Container = styled.header`
+  max-width: 1300px;
+  width: 80%;
+  margin: 0 auto;
+`;
+
+export const Nav = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 1.6em;
+
+  img{
+    display: block;
+  }
+`;
+
+const ListItem = styled.li`
+  text-decoration: none;
+`;
+
+const NavLinks = styled.ul`
+  list-style-type: none;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
Again, very clean code. Nice job. 

I would set the font-family in the global styles. I didn't run the project but I assume most of the elements have a default font now, right? This should be Montserrat though iirc

Reply:
This is the first time I am working with Styled-components. I forgot to put the fonts in GlobalStyles. Anyway, I'll make the changes now. Thank you.